src/index.ts

```diff
@@ -1,8 +1,15 @@
 import { cryptoCallbacks } from './crypto_callbacks';
 export { cryptoCallbacks };
 
-import bindings = require('bindings');
-const mc: MongoCryptBindings = bindings('mongocrypt');
+function load() {
+  try {
+    return require('../build/Release/mongocrypt.node');
+  } catch {
+    return require('../build/Debug/mongocrypt.node');
+  }
```

Comment:
I just realized this is the opposite order than I think is what we want right? because in production folks should always have "Release" so if that's missing then that's the error we should throw but currently we're always going to squash that with a "Debug not found" error. 

I'm not sure this matters too much, but just confirming we got the order right here? It's the opposite of what was suggested: https://mongodb.slack.com/archives/GGWBN4ZNK/p1731602768255219?thread_ts=1731521270.539249&cid=GGWBN4ZNK

Reply:
No, I intentionally choose Release first.  Why would we try to load Debug in production?  Debug is the special case, not Release, so we should fall back to the special case.

I don't think it matters much, I don't expect users to see this error.  If you feel strongly, we can catch the error and rethrow it, squashing the debug error.  Or throw an aggregate error or something.

Reply:
Fine with it as is just confirming here